web/07-continue.php

```diff
@@ -73,10 +73,15 @@
 }
 
 # start the workflow again
-chdir($folder);
-pclose(popen("$exec &", 'r'));
+if ($rabbitmq_host != "") {
+  $msg_exec = str_replace("\"", "'", $exec);
+  $message = '{"folder": "' . $folder . '", "custom_application": "' . $msg_exec . '"}';
```

Comment:
problem with this is that I can send now arbitrary code to the container to be executed. I'd rather have flags that we can pass along and use these flags to do specific things in the executor container.

Reply:
I get where you're coming from, but I'm tempted to punt on it because that fix is a bit more involved. Also, the current system _already_ allows you to execute any arbitrary code on the containers via a custom `job.sh` script. IIRC, we can even edit that directly from the web interface, via "Edit Model Configs".

Reply:
Case in point:

<img width="1251" alt="screen shot 2018-10-18 at 9 42 10 am" src="https://user-images.githubusercontent.com/5680221/47158870-96c45780-d2ba-11e8-961f-d6b66f5e1e10.png">

Reply:
yeah, we should figure out how to fix that :) maybe only allow edit if logged in.